google/cloud/internal/common_options.h

```diff
@@ -0,0 +1,68 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_INTERNAL_COMMON_OPTIONS_H
+#define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_INTERNAL_COMMON_OPTIONS_H
+
+#include "google/cloud/internal/options.h"
+#include "google/cloud/version.h"
+#include <set>
+#include <string>
+
+namespace google {
+namespace cloud {
+inline namespace GOOGLE_CLOUD_CPP_NS {
+
+/**
+ * Change the endpoint.
+ *
+ * In almost all cases the a suitable default will be chosen automatically.
+ * Applications may need to be changed to (1) test against a fake or simulator,
+ * or (2) to use a beta or EAP version of the service.
```

Comment:
"changed to (1) test ..., or (2) use a ..."

OR

"changed (1) to test ..., or (2) to use a ..."